tools/test

```diff
@@ -179,6 +214,17 @@ run_jest() {
             ;;
     esac
 
+    case "$platform" in
+        ios) jest_args+=( --selectProjects ios );;
+        android) jest_args+=( --selectProjects android );;
+        both) jest_args+=( --selectProjects ios android );;
+
+        # This is where `sloppy` is sloppy: we just choose iOS, so the
+        # tests will run faster, but at the expense of a (relatively
+        # small) loss of coverage.
+        sloppy) jest_args+=( --selectProjects ios );;
```

Comment:
A possibly dumb idea — we could pick whether it runs as iOS or Android randomly. The idea there being that if it's only failing on one, picking randomly would cause you to notice sooner during local development, rather than it only being caught in CI. And while that makes it flaky, it's a pretty reasonable type of flake, since it'll always be catching a real test failure.

Reply:
Interesting; I kind of like this idea. @gnprice, what do you think?

Reply:
Hmm, maybe. I like the idea of causing you to notice sooner. I think this would definitely best be done as a separate change.